Tests/BinaryParsingTests/IntegerParsingTests.swift

```diff
@@ -621,4 +639,20 @@ struct IntegerParsingTests {
     try fuzzIntegerCasting(
       UInt.self, loadingFrom: UInt64.self, using: &rng)
   }
+  
+  // Some LEB128 encoders output padding bytes which are considered
+  // valid if the number of bytes does not exceed `ceil(bitWidth / 7)`.
+  @Test(arguments: [
```

Comment:
Just some whitespace issues left! If you want to keep this indentation, you can add this line, and then run the `Scripts/format.sh` script to do the formatting locally.

```suggestion
  // swift-format-ignore
  @Test(arguments: [
```

Reply:
Thanks I applied the patch instead, I'm happy to let the machines rule the whitespace!